components/suggest/src/schema.rs

```diff
@@ -121,6 +123,12 @@ pub const SQL: &str = "
         description TEXT NOT NULL,
         FOREIGN KEY(suggestion_id) REFERENCES suggestions(id) ON DELETE CASCADE
     );
+
+    -- Just store the MD5 hash of the dismissed suggestion.  The collision rate is low and the
+    -- impact of a collision is not showing a suggestion, which is not that bad.
+    CREATE TABLE dismissed_suggestions (
+        url_hash INTEGER PRIMARY KEY
```

Comment:
This looks good, just a note that we might want to use `INSERT OR IGNORE` to avoid spurious insert failures due to hash collisions.